server/server/src/config.rs

```diff
@@ -50,6 +50,23 @@ impl FilesDbConfig {
     }
 }
 
+#[derive(Clone)]
+pub struct StripeConfig {
+    pub stripe_secret: String,
+    pub signing_secret: String,
+    pub premium_price_id: String,
```

Comment:
this idea in this PR is called 3 different things:
+ `Monthly` in tier selection
+ `PAID` in the constant in `file_index_repo`
+ `premium` here


You should pick one and unify all of these.